UtilityApps/CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ dd4hep_add_executable( print_materials src/print_materials.cpp USES DDRec )
 #-----------------------------------------------------------------------------------
 dd4hep_add_executable( materialScan    src/materialScan.cpp USES DDRec )
 #-----------------------------------------------------------------------------------
+dd4hep_add_executable( graphicalMaterialScan src/graphicalMaterialScan.cpp USES DDRec )
```

Comment:
Thanks for the feedback. Excuse my ignorance about the workflow: is this a request for me to fix it, or a notification that you have done so?